### PR TITLE
Update use-content-search-for-targeted-collections.md

### PR DESCRIPTION
**Pull request by shelri**
https://o365exchange.visualstudio.com/Enterprise%20Cloud/_workitems/edit/2159632/

Trailing asterisk is required or the user will not return any results.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/use-content-search-for-targeted-collections.md [hunk 1]
```diff
@@ -254,13 +256,13 @@ Here are some examples of using the  `folderid` and  `documentlink` properties i
 - This example searches a site folder (and any subfolders) for documents that contain the letters "NDA" in the title.
 
   ```powershell
-  documentlink:<path> AND filename:nda
+  documentlink:"<path>/*" AND filename:nda
   ```
 
 - This example searches a site folder (and any subfolder) for documents there were changed within a date range.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/use-content-search-for-targeted-collections.md:262]:
```suggestion
- This example searches a site folder (and any subfolder) for documents that were changed within a date range.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @shelri, not related to your PR but could you please also commit my suggested change to fix the typo. Many thanks.

**Comment by markjjo** [comment]
@shelri PR merged. Thanks for adding this bit.

File: microsoft-365/compliance/use-content-search-for-targeted-collections.md (hunk 1 above)
Comment (shelri) [reply microsoft-365/compliance/use-content-search-for-targeted-collections.md:262]: Will do

**Pull request merged**